Makefile.dep

```diff
@@ -12,6 +12,31 @@ ifneq (,$(filter nhdp,$(USEMODULE)))
   USEMODULE += oonf_rfc5444
 endif
 
+ifneq (,$(filter gnrc_netif_default,$(USEMODULE)))
+  USEMODULE += gnrc_netif
+  USEMODULE += netif_default
```

Comment:
No, that is why it doesn't make any sense at the board level. That's what I'm saying. But it DOES make sense to give the user the option to say

```Makefile
USEMODULE += gnrc_netdev_default
```

instead of

```Makefile
USEMODULE += netdev_default
USEMODULE += gnrc_netdev2 # or whatever gnrc_netdev requires
```

at the application level.